Dalamud/Game/Gui/Dtr/DtrBarEntry.cs

```diff
@@ -68,7 +73,7 @@ public interface IDtrBarEntry : IReadOnlyDtrBarEntry
     /// <summary>
     /// Gets or sets an action to be invoked when the user clicks on the dtr entry.
     /// </summary>
-    public Action<AddonMouseEventData>? OnClick { get; set; }
+    public new Action<DtrInteractionEvent>? OnClick { get; set; }
```

Comment:
Does the readonly interface still exist? If so, would you mind adding this to that one as well?

Reply:
See above: lines 46-50. OnClick has `get` only. Does that satisfy?

Reply:
Oh, apologies. I didn't realize that was the interface. Yes, this is great! <3